### PR TITLE
Add static error pages

### PR DESCRIPTION
**Pull request by jfranciscosousa**

### CODE REVIEW

File: public/404.html [hunk 1]
```diff
@@ -1,67 +1,49 @@
...
+
+        font-size: 20px;
+        font-weight: bold;
+        line-height: 28px;
+      }
+    </style>
+  </head>
+
+  <body class="rails-default-error-page">
+    <!-- This file lives in public/500.html -->
```
Comment (frm) [inline public/404.html]: 404 mate

File: public/422.html [hunk 2]
```diff
@@ -1,67 +1,49 @@
...
+
+        font-size: 20px;
+        font-weight: bold;
+        line-height: 28px;
+      }
+    </style>
+  </head>
+
+  <body class="rails-default-error-page">
+    <!-- This file lives in public/500.html -->
```
Comment (frm) [inline public/422.html]: 422. Also, should we use the same error for 500 and 422?

File: public/404.html (hunk 1 above)
Comment (jfranciscosousa) [reply public/404.html]: Copy pasted

File: public/422.html (hunk 2 above)
Comment (jfranciscosousa) [reply public/422.html]: Idk, for the end user, it's irrelevant. For us? We have Sentry so... we always know what's up

**Comment by frm** [comment]
Does the `favicon.ico` belong here?

**Comment by jfranciscosousa** [comment]
@justmendes I'll remove the favicon.

**Review by frm — APPROVED** [review]

**Pull request merged**